frontend/app-development/layout/PageHeader/SubHeader/SettingsModalButton/SettingsModal/components/Tabs/Maskinporten/ScopeList/utils.ts

```diff
@@ -0,0 +1,87 @@
+import { type StudioCheckboxTableRowElement } from '@studio/components';
+import { type MaskinportenScope } from 'app-shared/types/MaskinportenScope';
+
+export const mapScopesToRowElements = (
+  maskinPortenScopes: MaskinportenScope[],
+  selectedScopes: MaskinportenScope[],
+): StudioCheckboxTableRowElement[] => {
+  const allScopes = mergeTwoScopeListsToOne(maskinPortenScopes, selectedScopes);
```

Comment:
Should we rename to be more accurate? 😊 

```suggestion
  const maskinportenScopes = mergeTwoScopeListsToOne(maskinPortenScopes, selectedScopes);
```

Reply:
Don't think that will be ideal because the scopes sent in on line 5 is also named `maskinportenScopes`. 
I can rename it to `combinedScopesList` or something? 🤔